downstream/titles/automation-mesh/docinfo.xml

```diff
@@ -1,4 +1,4 @@
-<title>Red Hat Ansible Automation Platform Automation Mesh Guide for VM-based installations</title>
+<title>Red Hat Ansible Automation Platform Automation mesh guide for VM-based installations</title>
```

Comment:
I believe Automation should be lower case too, so:
Red Hat Ansible Automation Platform automation mesh guide for VM-based installations